internal/testutil/servicetest/authz.go

```diff
@@ -0,0 +1,40 @@
+package servicetest
+
+import (
+	"context"
+
+	"clouditor.io/clouditor/internal/api"
+	"clouditor.io/clouditor/service"
```

Comment:
Shouldn't we try to completely avoid dependencies from the service packages? If yes, we could move the `RequestType` to API?

Reply:
Hmm yeah this dependency might cause problems. I am not sure about moving `RequestType` to API, since it's something really only related to the service itself and not used by any of the api / "client" packages. If you really want to remove the dependency, I could move it to `internal/service` and add an alias in `service`, similar to what we do for stuff in `internal/api` / `api`, e.g., `api.PayloadRequest`?

Reply:
In the end, I moved the type to the api package, it really seems to be cleaner.

Reply:
ah! but authorisation strategy is still in the `service` package and definitely belongs there ... So I cannot remove this dependency.

Reply:
I would keep this dependency for now. I think somewhere this year we probably need a more or less complete overhaul of the package structure, because we never really had a good structure in mind and just kept putting things there where they fit and it seems to cause us problems in some of the tests.